recipes/geomet/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "geomet" %}
+{% set version = "0.2.0.post2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f5ea069d01afecaab0c66a1ca516e2716a64aeecfa65021d15bc3b1d9aaa35e4"
+
+build:
+  number: 0
+  entry_points:
+    - geomet=geomet.tool:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [py<32]
+
+requirements:
+  host:
+    - click
+    - pip
+    - python
+    - six
+  run:
+    - click
+    - python
+    - six
+
+test:
+  imports:
+    - geomet
+  commands:
+    - geomet --help
+
+about:
+  home: "https://github.com/geomet/geomet"
+  license: "Apache Software"
```

Can you use the SPDX identifiers:
```suggestion
  license: Apache-2.0
```
(I assume it is 2.0)